projects/project.cpp

```diff
@@ -598,6 +598,13 @@ namespace projects {
             bool shouldRefine {(r2 < r_max2) && ((P::useAlpha1 ? cell->parameters[CellParams::AMR_ALPHA1] > P::alpha1RefineThreshold : false) || (P::useAlpha2 ? cell->parameters[CellParams::AMR_ALPHA2] > P::alpha2RefineThreshold : false))};
             bool shouldUnrefine {(r2 > r_max2) || ((P::useAlpha1 ? cell->parameters[CellParams::AMR_ALPHA1] < P::alpha1CoarsenThreshold : true) && (P::useAlpha2 ? cell->parameters[CellParams::AMR_ALPHA2] < P::alpha2CoarsenThreshold : true))};
 
+            // Pressure anisotropy forces refinement
```

Comment:
Please elaborate (low pressure anisotropy; when below threshold, assume some kind of beam which will require higher resolution)

Reply:
This is documented in the `parameters.cpp` description as it should. As for the scientific motivation, I guess that could be added there as well but I wouldn't be the one to ask about that :grin: